recipes/pulseaudio/all/conanfile.py

```diff
@@ -122,12 +122,34 @@ def package(self):
         tools.remove_files_by_mask(self.package_folder, "*.la")
 
     def package_info(self):
-        self.cpp_info.libdirs = ["lib", os.path.join("lib", "pulseaudio")]
-        if self.options.with_glib:
-            self.cpp_info.libs.append("pulse-mainloop-glib")
-        self.cpp_info.libs.extend(["pulse-simple", "pulse"])
+        self.cpp_info.components["pulse"].names["pkg_config"] = "libpulse"
+        self.cpp_info.components["pulse"].libs = ["pulse"]
         if not self.options.shared:
-            self.cpp_info.libs.append("pulsecommon-%s" % self.version)
-        self.cpp_info.defines = ["_REENTRANT"]
-        self.cpp_info.names["pkg_config"] = "libpulse"
+            self.cpp_info.components["pulse"].libs.append("pulsecommon-%s" % self.version)
+        self.cpp_info.components["pulse"].libdirs = ["lib", os.path.join("lib", "pulseaudio")]
+        self.cpp_info.components["pulse"].requires = ["libsndfile::libsndfile", "libcap::libcap"]
+        if self.options.with_alsa:
+            self.cpp_info.components["pulse"].requires.append("libalsa::libalsa")
+        if self.options.with_glib:
+            self.cpp_info.components["pulse"].requires.append("glib::glib")
```

Comment:
https://gitlab.freedesktop.org/pulseaudio/pulseaudio/-/blob/master/libpulse.pc.in it looks to me that this component should not link to glib